tests/i18n/test_extraction.py

```diff
@@ -638,6 +638,31 @@ def test_default_root_settings(self):
         _, po_contents = self._run_makemessages(domain="djangojs")
         self.assertMsgId("Static content inside app should be included.", po_contents)
 
+    def test_i18n_catalog_ignored_when_invoked_for_django(self):
+        """
+        Regression test for #34848.
+        """
+        # Ensure "invoked_for_django" is True
+        os.makedirs(os.path.join("conf", "locale"))
+        # Create target file so it's actually ignored
+        i18n_catalog_js_dir = os.path.join(os.path.curdir, "views", "templates")
```

Comment:
To add more explanation, comments should state the expected behavior and omit prefixes like "Ensure" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Maybe:
```suggestion
        # Create a target file in the directory recognized as
        # "invoked_for_django" to be ignored.
        os.makedirs(os.path.join("conf", "locale"))
        i18n_catalog_js_dir = os.path.join(os.path.curdir, "views", "templates")
```